core/domain/exp_jobs_one_off.py

```diff
@@ -57,6 +60,32 @@ def map(exploration_model):
     def reduce(exp_id, list_of_exps):
         pass
 
+class ExpSummariesContributorsOneOffJob(jobs.BaseMapReduceJobManager):
+    """ One-off job that finds the user id's of the contributors
+    (defined as any human who has made a positive (i.e. not just
+    a reversion) for each exploration.
+    """
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationSnapshotMetadataModel]
+
+    @staticmethod
+    def map(item):
+        _VERSION_DELIMITER = '-'
```

Comment:
Hey @seanlip , could you point me to other places in the code base where a method is created on an item in a model? I've seen cases where say the versioned id would be passed in, and we'd pass back the unversioned one, but not sure where to look for examples of per-tuple methods-- let me know if that question doesn't make sense, thanks!